app/config/settings.py

```diff
@@ -28,8 +28,8 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = os.environ.get("DEBUG", "True") != "False"
 
-ALLOWED_HOSTS = os.environ.get("DJANGO_ALLOWED_HOSTS", "localhost").split(" ")
-
+# ALLOWED_HOSTS = os.environ.get("DJANGO_ALLOWED_HOSTS", "localhost").split(" ")
+ALLOWED_HOSTS = ["*"]
```

Comment:
This isn't really safe. Can we switch to using the workaround?

Reply:
updated to inject ALLOWED_HOSTS with IP of ECS health checker